answerking_app/views/item_views.py

```diff
@@ -1,16 +1,16 @@
 import json
 from json import JSONDecodeError
 
-from django.db.models import QuerySet
-from django.http import HttpResponse, JsonResponse, HttpRequest
 from django.core.serializers.json import DjangoJSONEncoder
+from django.db.models import QuerySet
+from django.http import HttpRequest, HttpResponse, JsonResponse
```

Comment:
These will cause a clash because i have used Request, Response from rest_framework. just a heads up.

Reply:
yes, will be rebased once yours will be merged with develop